### PR TITLE
Build trees, no fancy features.

### PR DESCRIPTION
**Pull request by nsg**
@petterarvidsson Can you show me how to keep the different actors/trees separate, like we talked about?

### CODE REVIEW

File: src/main/java/org/konstructs/forest/MakeTree.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import konstructs.ProbabilisticProductionRule;
+import konstructs.ProbalisticProduction;
+import konstructs.ProductionRule;
+import konstructs.Position;
+import konstructs.api.PutBlock;
+import java.util.Collection;
+
+public class MakeTree {
+
+    public MakeTree() {}
```
Comment (petterarvidsson) [inline src/main/java/org/konstructs/forest/MakeTree.java]: @nsg Remove the constructor

File: src/main/java/org/konstructs/forest/MakeTree.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    public Collection<PutBlock> buildTree(Position pos) {
```
Comment (petterarvidsson) [inline src/main/java/org/konstructs/forest/MakeTree.java]: And make this method static `public static ...`

Comment (nsg) [inline src/main/java/org/konstructs/forest/MakeTree.java]: ... or just refactor the class a little. It's just a copy-paste mostly atm.

File: src/main/java/org/konstructs/forest/ActorManager.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
+package org.konstructs.forest;
```
Comment (petterarvidsson) [inline src/main/java/org/konstructs/forest/ActorManager.java]: @nsg I think this base class looks very promising. I like the implemented onX methods, that means that one must not fully understand akka, while at the same time one must not implement the methods. Really nice, but one small thing can be improved. I think they should all call `unhandled(message);`, that way the message is also logged.

**Comment by nsg** [comment]
Old PR, old dead code

**Pull request closed without merging**